it_hardware/data/knowledge_article.xml

```diff
@@ -0,0 +1,215 @@
+                </p>
+                <div data-oe-protected="true" class="o_knowledge_behavior_anchor o_knowledge_behavior_type_toc">
+                    <div class="o_knowledge_toc_content">
+                        <a href="#" data-oe-nodeid="0" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_0">IT Hardware Sales &amp; Service</a>
+                        <a href="#" data-oe-nodeid="1" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_1"> Configuration</a>
+                        <a href="#" data-oe-nodeid="2" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_1">Business Flow&nbsp;</a>
+                        <a href="#" data-oe-nodeid="3" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_2">Flow 1: Buy Laptop &amp; Accessories</a>
+                        <a href="#" data-oe-nodeid="4" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_2">Flow 2: B2C eCommerce Sale</a>
+                        <a href="#" data-oe-nodeid="5" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_2">Flow 3:&nbsp;Bulk Order</a>
+                        <a href="#" data-oe-nodeid="6" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_2">Flow 4: Installation Service</a>
+                        <a href="#" data-oe-nodeid="7" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_2">Flow 5: Repair Service</a>
+                    </div>
+                </div>
+                <p><br/></p>
+                <h2>
+                    <span style="font-weight: bolder;">Configuration</span>
+                </h2>
+                <p style="margin-bottom: 0px;">
+                    <strong><font style="color: rgb(255, 156, 0);">Product Specification</font>:&nbsp;</strong>Laptops have attributes (Brand, CPU, ...) &amp; variants {RAM, HD)&nbsp;
+                </p>
+                <p style="margin-bottom: 0px;">
+                    <strong><font style="color: rgb(255, 156, 0);">Part Code:</font>&nbsp;</strong>Manage unique codes for IT components.
+                </p>
+                <p style="margin-bottom: 0px;">
+                    <strong><font style="color: rgb(255, 156, 0);">Lot &amp; Serial Numbers</font>:&nbsp;</strong>Track products with lot and serial numbers.
+                </p>
+                <p style="margin-bottom: 0px;">
+                    <strong><font style="color: rgb(255, 156, 0);">Warranty Days</font>:&nbsp;</strong>On products, if you enable "Expiration Date", you can set a warranty duration
+                </p>
+                <p><br/></p>
+                <h2>
+                    <span style="font-size: 24px;">Business Flow&nbsp;</span>
```

Comment:
```suggestion
                    <span style="font-size: 24px;">Business Flow</span>
```
Please remove `&nbsp;` from whole module and add ` ` where necessary

Reply:
@dhrs-odoo